test/parallel/test-fs-readfile.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const common = require('../common');
+
+// This test ensures that fs.readFile correctly returns the
+// contents of varying-sized files.
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const prefix = `.removeme-fs-readfile-${process.pid}`;
+
+common.refreshTmpDir();
+
+const fileInfo = [ { name: path.join(common.tmpDir, `${prefix}-1K.txt`),
+                     len: 1024,
+},
+                   { name: path.join(common.tmpDir, `${prefix}-64K.txt`),
+                     len: 64 * 1024,
+                   },
+                   { name: path.join(common.tmpDir, `${prefix}-64KLessOne.txt`),
+                     len: (64 * 1024) - 1,
+                   },
+                   { name: path.join(common.tmpDir, `${prefix}-1M.txt`),
+                     len: 1 * 1024 * 1024,
+                   },
+                   { name: path.join(common.tmpDir, `${prefix}-1MPlusOne.txt`),
+                     len: (1 * 1024 * 1024) + 1,
+                   },
```

Comment:
can you please align all of those to the left?

```js
const fileInfo = [{
}, {
}, {
}]
```

Reply:
Is cfa8576 acceptable?